actions/promote/createBatch.js

```diff
@@ -118,7 +118,7 @@ async function createBatch(batchManager, appConf) {
     return findAndBatchFGFiles({
         baseURI: sp.api.file.get.fgBaseURI,
         options,
-        fgFolders: ['/drafts'],
+        fgFolders: appConf.isDraftOnly() ? ['/drafts'] : ['/'],
```

Comment:
@raga-adbe-gh 
Should this be `['/']` or `['']` ?

When I had initially implemented it, the default was set to  `['']` which was further change to `['/drafts']` to restrict it as seen this in PR https://github.com/adobecom/milo-fg/pull/34/files

Reply:
Thanks @sukamat , You right this should be `['']` and this seems to work as expected.